scripts/test_tf2_execution.py

```diff
@@ -2,13 +2,15 @@
 import logging
 import os
 from pathlib import Path
-import itertools
+import itertools, contextlib
 
+import numpy as np
 import tensorflow as tf
 from tqdm import tqdm
 
 from checkmate.core.solvers.strategy_checkpoint_all import solve_checkpoint_all
 from checkmate.core.solvers.strategy_chen import solve_chen_sqrtn
+from checkmate.core.solvers.strategy_optimal_ilp import solve_ilp_gurobi
```

Comment:
Importing this file can only happen if Gurobi is installed. If you want to import gurobi, gate it with a check like https://github.com/parasj/checkmate/blob/b9f1d0c84e3bf8c71394c65c4e399e616c9d0864/tests/test_linear.py#L94-L100

Reply:
Will do